README.md

```diff
@@ -5,10 +5,18 @@
 
 ### minikube-action Arguments
 - minikube-version (optional)
-  - description: allows you to use a specific minikube version
+  - description: Choose a specific minikube version
```

Comment:
lets add a full example also in addition to this doc

as example 4
with configuring everyghing that this setup-minikube allows u to config

Reply:
Example 2 contains full configuration
```
      with:
        minikube-version: 1.24.0
        driver: docker
        container-runtime: containerd
```